packages/client/test/sim/configs/prysm/txGenerator.ts

```diff
@@ -0,0 +1,178 @@
+// Adapted from - https://github.com/Inphi/eip4844-interop/blob/master/blob_tx_generator/blob.js
+import { BlobEIP4844Transaction, initKZG } from '@ethereumjs/tx'
```

Comment:
should we just import the txgenerator from one level above or this has some customiizations?

Reply:
I modified the chain ID (and maybe something else but don't remember precisely).  I can work to make this a more generalized blob tx generator script that takes CLI args for all of the fields that would change if that's helpful.